zingolib/src/lightclient/propose.rs

```diff
@@ -150,10 +159,17 @@ impl LightClient {
         )
         .map_err(DoProposeError::Proposal)?;
 
-        let mut latest_proposal_lock = self.latest_proposal.write().await;
-        *latest_proposal_lock = Some(crate::data::proposal::ZingoProposal::Transfer(
-            proposal.clone(),
-        ));
+        Ok(proposal)
+    }
+
+    /// Unstable function to expose the zip317 interface for development
+    pub async fn do_propose_send_and_store(
```

Comment:
Hmm...   I don't know about the `do_` prefixes..  but how about:

`do_send_and_store_proposal`?

`send_and_store_proposal` sounds good to me...    how locked into, and what benefit do we get from `do_` in front of every public API method?